images/toolchain-aarch64-linux-gnu/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM debian:stable
 
+ARG  MAJOR
```

Comment:
`MINOR` and `DATE` may be also different for different releases, so we should make them arguments too.